Specs/Scene/ExpressionSpec.js

```diff
@@ -988,6 +988,78 @@ defineSuite([
         }).toThrowDeveloperError();
     });
 
+    it('evaluates atan2 function', function() {
+        var expression = new Expression('atan2(0,1)');
+        expect(expression.evaluate(frameState, undefined)).toEqual(0.0);
+
+        var expression = new Expression('atan2(1,0)');
+        expect(expression.evaluate(frameState, undefined)).toEqual(0.5*PI);
```

Comment:
If this test fails you may need to use `toEqualEpsilon` like here: https://github.com/AnalyticalGraphicsInc/cesium/pull/4628#issuecomment-261650132.

`PI` should be `Math.PI`